KsApi/Service+RequestHelpers.swift

```diff
@@ -43,45 +43,23 @@ extension Service {
       .map { json in decode(json) as M? }
   }
 
-  private func performRequest<A: Swift.Decodable>(request: URLRequest) -> SignalProducer<A, GraphError> {
```

Comment:
The `performRequest` function has been removed in favor of a `decodeGraphModel` function that decodes `Data` into objects, and an `rac_graphDataResponse` function which is responsible for actually making the request and parsing any resulting errors.